cmd/geth/flag.go

```diff
@@ -155,6 +155,12 @@ func mustMakeChainIdentity(ctx *cli.Context) (identity string) {
 		cacheChainIdentity = identity
 	}()
 
+	if chainFlagVal := ctx.GlobalString(aliasableName(ChainIdentityFlag.Name, ctx)); chainFlagVal != "" && strings.HasSuffix(chainFlagVal, "/") || strings.HasSuffix(chainFlagVal, "\\") {
+		suff := string(chainFlagVal[len(chainFlagVal)-1])
+		chainFlagVal = strings.Replace(chainFlagVal, suff, "", 1)
```

Comment:
Can't we just "chop" the string, remove last character, instead of doing `strings.Replace`, like:
`chainFlagVal = chainFlagVal[:len(chainFlagVal)-1]` (then we could even inline this in following line)?
(And I you decide to do `Clean` as mentioned above, the problem won't exist anyways).